### ISSUE TITLE
status?

### ISSUE THREAD

**Issue by aquynh** [thlorenz/visulator#1]
this looks amazing, except that i dont know how to play with it at the link https://thlorenz.github.io/visulator

is this not ready yet? hope to see more documentation soon!

i can send in a pull request to add the instructions on installing Capstone engine if you want it.

cheers.

**Comment by thlorenz** [comment]
It's not ready yet, capstone is not what's holding me back.
I'm currently working on handling [more opcodes](https://github.com/thlorenz/visulator/blob/master/lib/x86/cu.js#L162) before I'll move on to work more on the actual visualization in the browser.

I'm hoping to at least be able to run a small routine, before that a visualization won't make much sense IMO.

Follow me on twitter @thlorenz if you want. I'll announce there once I hit another milestone.

**Comment by aquynh** [comment]
yes just followed you. cant wait for the update on this project!
